styleguide/_themes/derek/scss/patterns/grid/productFeatures.scss

```diff
@@ -30,6 +31,17 @@
   }
 }
 
+.subpanel {
+  .features-container {
+    @include make-md-column(12);
+    margin: 5px 0;
+
+    .rsweb {
```

Comment:
We probably need a follow up task to refactor the features as they aren't using maintainable css. Until that happens, we need a way to distinguish between `.rsweb` inside of features, vs inside of other patterns, inside of subpanels. I'm not a fan of this, by any means, and I guess could just refactor this one portion if you'd prefer, but given that this will need follow up attention anyway, I'm wondering if it makes sense to just clean it up all at once?